styles.css

```diff
@@ -141,6 +139,11 @@ body {
   display: block;
 }
 
+.tonic:hover {
+  transform: rotate(360deg);
+  transition: all 0.3s ease-in-out 0s;
+}
+
 .header-tonic {
   width: 18.438rem;
   height: 3.5rem;
```

Comment:
- [ ] Your `header` is missing just one background image variant to make it perfect work. Kindly add it u to complement your good work and make the project stick to the requirements:

## Figma View:
![image](https://user-images.githubusercontent.com/50990925/222940441-a37ee273-e9b8-4838-af38-f65248355b80.png)

## Your view
![image](https://user-images.githubusercontent.com/50990925/222940458-860fef7c-f761-4681-befe-ac013211c283.png)